lib/vdsm/common/dsaversion.py.in

```diff
@@ -48,7 +48,7 @@ def version_info():
     cluster_levels = ['4.2', '4.3', '4.4', '4.5', '4.6']
     libvirt_version = _get_libvirt_version()
 
-    if libvirt_version >= (7, 10):
+    if libvirt_version >= (8, 0):
```

Comment:
4.7 cluster is not yet finalized, we are still waiting for final libvirt (which we hopefully have now) and final qemu (current one still doersn't have correct machine types). That's why cl 4.7 is not yet the default in oVirt 4.5

It seems to be better to rely on the version which released (and tested by OST) rather than some lower one.

Anyway we need to wait a few days with merging this patch until this libvirt will get to CS8 and CS9